### DISCUSSION TITLE
Github Desktop Logout

### DISCUSSION THREAD

**Discussion by jty-computation** [community/community#23462] [New to GitHub]
<p>Hello!<br>
I recently installed GitHub Desktop on my MacBook (M1). Everything seems to be working, but I can’t seem to figure out how to log out in order to sign on with a different account of mine.<br>
Any help is appreciated.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>maybe this one</p>
<p><a href="https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/configuring-basic-settings">https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/configuring-basic-settings</a></p>

**Comment by byrneh** [comment]
<p><a class="mention" href="/jty-computation">@jty-computation</a> this is possible from top menu bar by selecting select File then Options and for the  Accounts section you will see <a href="http://Github.com" rel="noopener nofollow ugc">Github.com</a>  and Enterprise Sign out and Sign in options.</p>

**Comment by jty-computation** [comment]
<p>This worked. Specifically, clicking “Github Desktop” then looking in “preferences”, and signing out under the accounts page.<br>
Thanks!</p>

**Comment by ettaboyle** [comment]
<p>Thank for closing the loop <a class="mention" href="/jty-computation">@jty-computation</a> :pray: Welcome to the forums! Great to see your first post was solved :slight_smile: Please pay it forward and help others if you can!</p>
<p>Have a great day!</p>

**Reply by dp40255-star** [reply]
Hello I am bignner                                                                                                 can you help me                                                                                                         I know HTML,CSS,JS                                                                                                    my name Rajat Prajapati                                                                                         I am 12 year old                                                                                                        coll me 9424428948
